### PR TITLE
Nc news 2

### PR DESCRIPTION
**Pull request by steelfish1010**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const res = require("express/lib/response");
```
Comment (Rosem10) [inline app.js]: This line can be removed, please!

File: models/topics.model.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const db = require("../db/connection");
+
+exports.sendTopics = () => {
```
Comment (Rosem10) [inline models/topics.model.js]: I'd suggest renaming this function to something  like 'fetch topics' as it should describe the interaction it's having with the database. If you consider how our model works - it goes into the database, retrieves the data we need, and then passes it up to the controller which THEN sends the information that we have selected.

**Review by Rosem10 — CHANGES_REQUESTED** [review]
Please look at changing that function name!

File: app.js (hunk 1 above)
Comment (steelfish1010) [reply app.js]: Removed! :)

File: models/topics.model.js (hunk 2 above)
Comment (steelfish1010) [reply models/topics.model.js]: Thanks, renamed to fetchTopics

**Comment by HeatherEver** [comment]
Looks good with those changes 👍

**Pull request merged**